api/src/pcapi/routes/backoffice_v3/auth.py

```diff
@@ -70,35 +68,26 @@ def authorize() -> utils.BackofficeResponse:
         )
         return werkzeug.exceptions.Forbidden()
 
-    if settings.IS_TESTING or settings.IS_DEV or settings.IS_INTEGRATION:
-        if not user:
-            user = create_local_admin_user(
-                email=google_user["email"],
-                first_name=google_user["name"],
-                last_name=google_user["family_name"],
-                given_name=google_user["given_name"],
-            )
+    if not user:
+        user = create_local_admin_user(
+            email=google_user["email"],
+            first_name=google_user["given_name"],
+            last_name=google_user["family_name"],
+        )
 
-        backoffice_api.upsert_roles(user, list(perm_models.Roles))
-        db.session.commit()
-    else:
-        if not user:
-            logger.info(
-                "Failed authentication attempt",
-                extra={"identifier": google_email, "user": "not found", "avoid_current_user": True, "success": False},
-                technical_message_id="backoffice_v3.authorize",
-            )
-            session["google_email"] = google_email
-            return redirect(url_for(".user_not_found"))
+    roles = list(perm_models.Roles)
```

Comment:
```suggestion
    if settings.IS_TESTING or settings.IS_DEV or settings.IS_INTEGRATION:
        roles = list(perm_models.Roles)
```
Il est plus prudent de ne passer dans le code donnant tous les rôles que sur condition, plutôt qu'un `if not` qui limite après.